src/views/citation_editable.ts

```diff
@@ -71,6 +73,11 @@ export class CitationEditableView extends CitationView<EditableBlockProps> {
       }
     }
   }
+  public selectNode = () => {
+    this.dom.classList.add('ProseMirror-selectednode')
+    const trackModalActive = document.body.dataset.trackModal
```

Comment:
please see my comment in https://github.com/Atypon-OpenSource/manuscripts-article-editor/pull/517
tldr: we need use established way of communication in the app

Reply:
reverted, I have used eventListenr to hande the context menu visibility and the selection only to show the insertion popup.